eng/versioning/version_client.txt

```diff
@@ -104,7 +104,8 @@ com.azure:azure-core-tracing-opentelemetry-samples;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-core-version-tests;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-cosmos;4.75.0;4.76.0-beta.1
 com.azure:azure-cosmos-benchmark;4.0.1-beta.1;4.0.1-beta.1
-com.azure.cosmos.spark:azure-cosmos-spark_3_2-12;1.0.0-beta.1;1.0.0-beta.1
+com.azure.cosmos.spark:azure-cosmos-spark_3;0.0.1-beta.1;0.0.1-beta.1
```

Comment:
is this version intentional ?
I thought we start with 1.0.0-beta.1

Reply:
Intentional because it is never supposed to be published. These are projects with <packaging>pom</packaging> - just a way to share source code across multiple projects/jars.